packages/features/eventtypes/components/tabs/instant/InstantEventController.tsx

```diff
@@ -76,6 +76,10 @@ export default function InstantEventController({
   const [instantEventState, setInstantEventState] = useState<boolean>(eventType?.isInstantEvent ?? false);
   const formMethods = useFormContext<FormValues>();
 
+  const [parameters, setParameters] = useState<string[]>(
+    formMethods.getValues("instantMeetingParameters") || []
+  );
+
```

Comment:
I tried using `useFieldArray` but it had some issues and type errors

Reply:
What were the issues?

Reply:
- It was splitting the string to object like { 1: "I", 2: "N" , 3 :"D" }. 

and it's mentioned in the docs that useFieldArray doesn't support flat field arrays.

Reply:
🫠